hack/make/go.mk

```
@@ -21,12 +21,15 @@ go/format: go/fmt go/gci
 go/vet:
 	go vet -copylocks=false $(LINT_TARGET)
 
+go/wsl:
+	wsl -fix ./pkg/...
+
 ## Runs golangci-lint
 go/golangci:
 	golangci-lint run --build-tags "$(shell ./hack/build/create_go_build_tags.sh true)" --timeout 300s
 
 ## Runs all the linting tools
-go/lint: go/format go/vet go/golangci
+go/lint: prerequisites/go-linting go/format go/vet go/golangci
```

```suggestion
go/lint: go/wsl go/format go/vet go/golangci
```
no bueno?

I would only add `prerequisites/` commands in the `make prerequisites` global command

---

> Also added the linting tools install/update make target as a prerequisite to the make go/lint command, so we don't have to think about locally updating the linters. (it slows the command down a bit, but the golangci-lint is already dog slow, so it doesn't really matter)

---

when I run `make go/lint` I want to make sure that the output shows what the CI would show.

It happened to me several times that the local linter was outdated, everything passed locally, but the CI exploded

but if for popular demand we don't want it, then I can live without it 😅

---

Yep, now I know the pain. Sounds good to me too :+1: